src/main.d

```diff
@@ -643,18 +643,18 @@ void performSync(SyncEngine sync, string singleDirectory, bool downloadOnly, boo
 				// no single directory sync
 				if (uploadOnly){
 					// Upload Only of entire sync_dir
-					log.log("Syncing changes from local path only - NOT syncing data changes from OneDrive ...");
+					if (!monitor) log.log("Syncing changes from local path only - NOT syncing data changes from OneDrive ...");
 					sync.scanForDifferences(localPath);
 				} else {
 					// No upload only
 					if (localFirst) {
 						// sync local files first before downloading from OneDrive
-						log.log("Syncing changes from local path first before downloading changes from OneDrive ...");
+						if (!monitor) log.log("Syncing changes from local path first before downloading changes from OneDrive ...");
 						sync.scanForDifferences(localPath);
 						sync.applyDifferences();
 					} else {
 						// sync from OneDrive first before uploading files to OneDrive
-						log.log("Syncing changes from OneDrive ...");
+						if (!monitor) log.log("Syncing changes from OneDrive ...");
```

Comment:
That is the only change that I am not sure about... without this I don't see *any* activity in the log whether onedrive is alive or not ... Not sure what would be a nice solution though...

Reply:
Agreed - there should be at least a very minimal notification in non-verbose mode when using monitor to advise it *is* running

Reply:
But not every 45 seconds, which is the problem the original code has.

Reply:
@sweharris 
At 45 seconds yes - it would be too much, but also you could use monitor_interval to lessen the impact [https://github.com/abraunegg/onedrive/blob/master/README.md#monitor_interval](https://github.com/abraunegg/onedrive/blob/master/README.md#monitor_interval)

Reply:
But then it's less responsive to detecting Onedrive changes.